Trim outdated sections from the tuning guide (#3534)
theforeman/foreman-documentation: guides/common/modules/proc_manually-tuning-puma-workers-and-threads-count.adoc

Reviewer: That looks decent.

= Manually tuning Puma workers and threads count

If you decide not to rely on xref:Puma_Workers_and_Threads_Auto_Tuning_{context}[], you can apply custom numbers for these tunables.

> In the example below we are using 2 workers and 5 threads:

Reviewer 2: ```suggestion
The following example uses 2 workers and 5 threads:
```
I know that we are not supposed to attribute actions to non-humans, but "we" or "you" looks odd.

Author: The "we" style is used almost everywhere in the guide so I'd prefer to keep that out of scope.

Reviewer 2: Fine with me. I created a bug to update this guide to comply with our guidelines so that should catch it.
